pkg/kotsadm/minio.go

```diff
@@ -58,6 +65,13 @@ func ensureMinioStatefulset(deployOptions types.DeployOptions, clientset *kubern
 		if err != nil {
 			return errors.Wrap(err, "failed to create minio statefulset")
 		}
+
+		if deployOptions.UseHostNetwork {
```

Comment:
should this block be moved to after the 'failed to find kotsadm-minio statefulset' section?

that way if someone messes up and forgets to include the flag the first time, it'll still be applied
(of course, then we also need to handle the get/create logic instead of just unconditional create)